### PR TITLE
UMP-307615-Configured-the-EmbedProperties-by-EmbedConfig-Json-ReactWithCore

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,148 +1,76 @@
-# BoldBI Embedding React with ASP.NET Core Samples
+# Bold BI Embedded Sample in React with ASP.NET Core
```
Comment (vinothkumar1732) [inline README.md]: Change the folder name as React with ASP.NETCore

File: ReactwithASP.NETCore/WeatherForecast.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+namespace MyReactApp;
```
Comment (vinothkumar1732) [inline ReactwithASP.NETCore/WeatherForecast.cs]: Is this file needed for embedding?

Remove the unwanted files from project like csproj.user file, bin, obj and package-lock.json.

File: ReactwithASP.NETCore/MyReactApp.csproj [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <IsPackable>false</IsPackable>
+    <SpaRoot>ClientApp\</SpaRoot>
+    <DefaultItemExcludes>$(DefaultItemExcludes);$(SpaRoot)node_modules\**</DefaultItemExcludes>
+    <SpaProxyServerUrl>https://localhost:44456</SpaProxyServerUrl>
+    <SpaProxyLaunchCommand>npm start</SpaProxyLaunchCommand>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.SpaProxy" Version="7.0.5" />
```
Comment (vinothkumar1732) [inline ReactwithASP.NETCore/MyReactApp.csproj]: Is this SpaProxy package needed?

Comment (vinothkumar1732) [reply ReactwithASP.NETCore/MyReactApp.csproj]: Change this project page MyReactApp.csproj and make it as **ReactWithDOTNetCore.csproj**

File: ReactwithASP.NETCore/ClientApp/src/components/FetchData.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  constructor(props) {
+    super(props);
+    this.state = { forecasts: [], loading: true };
+  }
+
+  componentDidMount() {
+    this.populateWeatherData();
+  }
+
+  static renderForecastsTable(forecasts) {
```
Comment (vinothkumar1732) [inline ReactwithASP.NETCore/ClientApp/src/components/FetchData.js]: Is these default methods needed?

**Review by vinothkumar1732 — COMMENTED** [review]
Remove weatherforecast related files.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Content updated

File: ReactwithASP.NETCore/WeatherForecast.cs (hunk 2 above)
Comment (ghost) [reply ReactwithASP.NETCore/WeatherForecast.cs]: This file is not needed. Now removed this file, and unwanted file also removed.

File: ReactwithASP.NETCore/MyReactApp.csproj (hunk 3 above)
Comment (ghost) [reply ReactwithASP.NETCore/MyReactApp.csproj]: SpaProxy package is needed. Csproj project name also changed.

File: ReactwithASP.NETCore/ClientApp/src/components/FetchData.js (hunk 4 above)
Comment (ghost) [reply ReactwithASP.NETCore/ClientApp/src/components/FetchData.js]: This component folder is not needed, so removed this file for my end.

File: ASP.NET Core/BoldBI.Embed.Sample/Views/BoldBIEmbed/EmbedConfigErrorLog.cshtml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    var dlgDiv = $('<div id="sample_dialog" ></div>');
+    targetContainer.append(dlgDiv);
+    $('body').append(targetContainer);
+    var dialog = new window.ejs.popups.Dialog({
+        header: 'Error Message',
+        width: '500px',
+        isModal: true,
+        showCloseIcon: true,
+        target: document.getElementById('custom_dialog'),
+        content: '<div>To compile and run the project, an embed config file needs to be required. Please use the <a href="https://help.boldbi.com/cloud-bi/site-administration/embed-settings/" target="_blank">URL</a> to obtain the JSON file from the Bold BI server.</div>'
```
Comment (florenceliliana) [inline ASP.NET Core/BoldBI.Embed.Sample/Views/BoldBIEmbed/EmbedConfigErrorLog.cshtml:29]: content: '<div>To compile and run the project, an embed config file needs to be required. Please use the <a href="https://help.boldbi.com/cloud-bi/site-administration/embed-settings/" target="_blank">URL</a> to obtain the JSON file from the Bold BI server.</div>'.

File: README.md [hunk 6]
```diff
@@ -1,148 +1,75 @@
...
+  
+  5. To run the application, use the command `dotnet run` in the terminal. After executing the command, the application will automatically launch in the default browser. You can access it at the specified port number (e.g., https://localhost:5001/).
+    
+      ![dashboard image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
+
+Please refer to the [help documentation](https://help.boldbi.com/embedding-options/embedding-sdk/samples/react-core/#how-to-run-the-samples) to know how to run the sample.
+
+## Important notes
+
+It is recommended to not store passwords and sensitive information in configuration files for security reasons, in a real-world application. Instead, you should consider using a secure application, such as Key Vault, to safeguard your credentials.
```
Comment (florenceliliana) [inline README.md]: It is recommended not to store passwords and sensitive information in configuration files for security reasons in a real-world application. Instead, you should consider using a secure application, such as Key Vault, to safeguard your credentials.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: ASP.NET Core/BoldBI.Embed.Sample/Views/BoldBIEmbed/EmbedConfigErrorLog.cshtml (hunk 5 above)
Comment (ghost) [reply ASP.NET Core/BoldBI.Embed.Sample/Views/BoldBIEmbed/EmbedConfigErrorLog.cshtml:29]: Content updated

File: README.md (hunk 6 above)
Comment (ghost) [reply README.md]: Content updated

**Review by florenceliliana — APPROVED** [review]

**Review by Alagarsamyd — APPROVED** [review]
